README.md

```diff
@@ -13,87 +16,38 @@ The GenLayer prototype consists of the following main components:
 
 ## Installation
 
-### Window One
-
-```
-$ cp .env.example .env
-$ docker compose build
-$ docker compose up
-```
-
-### Window Two
-
-#### 1. Installing the Ollama model
-
-```
-$ docker exec -it ollama ollama run llama2
-...
-```
-
-#### 2. Set up the environment
-
-##### Linux / MacOS
-```
-$ virtualenv .venv
-$ source .venv/bin/activate
-(.venv) $ pip install -r requirements.txt
-(.venv) $ export PYTHONPATH="$(pwd)"
-```
-
-##### Windows (cmd)
 ```
-$ virtualenv .venv
-$  .\.venv\Scripts\activate
-(.venv) $ pip install -r requirements.txt
-(.venv) $ set PYTHONPATH=%cd%
+$ npm install -g genlayer
+$ genlayer init
```

Comment:
genlayer init to set it up for the first time

genlayer up to run it afterwards

Visit https://docs.genlayer.com/simulator/installation to learn more